app/views/catalog/_two_step_illiad_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="two-step-illiad" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+              <div class="modal-dialog" role="document">
+                <div class="modal-content">
+                  <div id="direct-link" class="modal-header">
+                    <h5 class="mb-0">Request Item</h5>
+                    <button type="button" class="blacklight-modal-close close" data-dismiss="modal" aria-label="<%= t('blacklight.modal.close') %>">
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                  </div>
+                  <div class="modal-body">
+                    <% physical_holdings.each do |holding| %>
```

Comment:
@alexBLR Related to my last comment: this should the array of doc_delivery_links we're iterating over, right? Because that list has been tested to see if they are approved for this specific user, and may be less than the list of physical holdings.